packages/ariakit/src/popover/popover-arrow.tsx

```diff
@@ -12,15 +12,16 @@ import { POPOVER_ARROW_PATH } from "./__popover-arrow-path";
 import { PopoverContext } from "./__utils";
 import { PopoverState } from "./popover-state";
 
+const defaultSize = 30;
+const halfDefaultSize = defaultSize / 2;
+
 const rotateMap = {
-  top: "rotate(180 0 0)",
-  right: "rotate(-90 0 0)",
-  bottom: "rotate(0 0 0)",
-  left: "rotate(90 0 0)",
+  top: `rotate(180 ${halfDefaultSize} ${halfDefaultSize})`,
+  right: `rotate(-90 ${halfDefaultSize} ${halfDefaultSize})`,
+  bottom: `rotate(0 ${halfDefaultSize} ${halfDefaultSize})`,
+  left: `rotate(90 ${halfDefaultSize} ${halfDefaultSize})`,
 };
```

Comment:
Thank you so much for researching and fixing this issue @SCasarotto! It works great.

I'm just trying to wrap my head around this `halfDefaultSize` logic. I can't see the difference between `rotate(180 15 15)` and `rotate(180)`. Could you add a comment explaining that?

I wonder if we could just do `rotate(180)`, `rotate(-90)` etc.

Reply:
Oh, I see now the difference in `right` and `left`.

Reply:
When I try that on Chrome I see this:
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/13336976/154826053-2386eae8-6e52-4330-83dd-1d7c413e8f29.png">

Am I missing something?

Reply:
The halfDefaultSize moves the rotation origin to the center. If its not centered it rotates about it 0,0 point. 

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#rotate

Reply:
I see! That makes sense. That works even if we pass a different `size` prop to the `PopoverArrow` component, right?

Reply:
> When I try that on Chrome I see this: <img alt="image" width="1251" src="https://user-images.githubusercontent.com/13336976/154826053-2386eae8-6e52-4330-83dd-1d7c413e8f29.png">
> 
> Am I missing something?

You're right! I can see that as well.